fee-hook/README.md

```diff
@@ -0,0 +1,41 @@
+# Fee Hook
+
+## Design
+
+- Loosely based on [Uniswap v4 Hooks template](https://github.com/uniswapfoundation/v4-template).
+- Fee Hook uses [uniswap v4 hooks](https://docs.uniswap.org/contracts/v4/concepts/hooks) to automatically collect liquidity fees and distribute them to vMOONEY holders.
+- Pools are registered with the _beforeAddLiquidity callback.
+- After swaps, fees are automatically withdrawn once a threshold is hit.
```

Comment:
Do fees go to a contract where people can then go an "grab" them, or do they automatically show up in the user's wallet?

Reply:
the accumulate in the contract, then when a user wants to withdraw they call withdrawFees()